R/pkg/R/mllib.R

```diff
@@ -694,8 +694,14 @@ setMethod("predict", signature(object = "KMeansModel"),
 #' }
 #' @note spark.mlp since 2.1.0
 setMethod("spark.mlp", signature(data = "SparkDataFrame"),
-          function(data, blockSize = 128, layers = c(3, 5, 2), solver = "l-bfgs", maxIter = 100,
-                   tol = 0.5, stepSize = 1, seed = 1) {
+          function(data, layers, blockSize = 128, solver = "l-bfgs", maxIter = 100,
+                   tol = 1E-6, stepSize = 0.03, seed = 0x7FFFFFFF) {
+            if (length(layers) <= 1) {
+              stop("layers vector require length > 0.")
+            }
+            if (any(sapply(layers, function(e) !is.numeric(e)))) {
```

Comment:
oh, its a clever way using `as.intege(x) != x` to check whether it is an integer.
here the mlp require layers to be integer vector,
is it better to force user pass integer vector, if not call `stop`, or just print a warning ?